packages/components/src/Slider/Slider.component.js

```diff
@@ -47,6 +47,29 @@ export function getCaptionsValue(captionsLength, min, max) {
 	return captionsValue;
 }
 
+/**
+ * Return correct modifier styles for usage mode
+ * @param {string} mode
+ * @param {number} value
+ */
+export function getModeStyles(mode, value) {
+	if (Array.isArray(value)) {
+		if (mode === 'exclusive') {
+			return 'exclusive';
+		} else {
+			return false;
+		}
+	} else {
+		if (mode === 'greaterThan') {
+			return 'greaterThan';
+		} else if (mode === 'equals') {
+			return 'equals';
+		} else {
+			return false;
+		}
+	}
+}
```

Comment:
Not sure what this function is for ?
Can't you just use the mode value to set the classname ?

Reply:
Something like `SLIDER_MODE` could be used alone but wouldn't stop someone calling a range slider and applying `equals` styling which would probably not be wanted. I don't know if we need to worry about protecting against this. If not then it can be simpler. 

We could alternatively put this protection in the CSS if we give the slider a .range class and use `:not`

Reply:
I don't think this is necessary, it adds a level of complexity, to try to prevent a development usage error. But the dev error will be directly visible.
What do you think ?

Reply:
I agree. I'll refactor this and maybe just document in the stories that `inclusive/exclusive` are valid for range sliders.